INSTALL.rst

```diff
@@ -21,13 +21,20 @@ Then, the rest of the packages can be installed via:
 
 ::
 
-    yum install libvirt bridge-utils ipmitool telnet sqlite httpd mod_wsgi python-pip qemu-kvm python-virtinst
+    yum install libvirt bridge-utils ipmitool telnet httpd mod_wsgi python-pip qemu-kvm python-virtinst
 
 In addition, HaaS depends on a number of python libraries. Many of these are
 available as RPMs as well, but we recommend installing them with pip, since
 this will install the versions that HaaS has been tested with.  This is done
 automatically by the instructions below.
 
+HaaS supports both Sqlite and Postgresql. To install HaaS using Postgresql you need to setup
+Postgresql. You can find the details in HACKING.rst. But if you want to install HaaS using 
```

Comment:
It probably makes more sense to have the instructions for the production-oriented database here, and put a reference to it in HACKING than the other way around.